### PR TITLE
Add REST service

### PR DESCRIPTION
**Pull request by cianclarke**

### CODE REVIEW

**Comment by lfryc** [comment]
FHCAP PR: https://github.com/fheng/fhcap/pull/1065

File: global.json [hunk 1]
```diff
@@ -1021,6 +1021,30 @@
...
+        "image": "public/img/cloud_plugins/fh.png",
+        "category": "Platform Service",
+        "description": "Explore APIs which may be protected by a firewall, or only accessible to the cloud via VPN. Supports saving requests, and generates useful code snippets.",
+        "appTemplates": [
+          {
+            "id": "rest-client-service",
+            "name": "Barcode Reader",
+            "type": "cloud_nodejs",
+            "repoUrl": "git://github.com/feedhenry-templates/fh-api-mapper.git",
+            "githubUrl": "https://github.com/feedhenry-templates/barcode-reader-service.git",
```
Comment (lfryc) [inline global.json]: this seems like copy&paste error, and may be cause of a failure to deploy this service, details here: https://github.com/fheng/fhcap/pull/1065#issuecomment-143977481

Comment (cianclarke) [inline global.json]: Yikes - nice catch @lfryc, thanks! Fixed

**Comment by lfryc** [comment]
review, works fine

**Pull request merged**